python/tvm/relay/op/strategy/x86.py

```diff
@@ -591,18 +591,17 @@ def dense_strategy_cpu(attrs, inputs, out_type, target):
 def dense_pack_strategy_cpu(attrs, inputs, out_type, target):
     """dense_pack x86 strategy"""
     strategy = _op.OpStrategy()
-
     if (
         inputs[0].dtype == "uint8"
         and inputs[1].dtype == "int8"
         and out_type.dtype == "int32"
         and attrs["weight_layout"] == "NC16n4c"
     ):
```

Comment:
Let's check the target mcpu here and pass VNNI or AMX attribute to `dense_int8`.

Reply:
@masahi already added the target checking, seems generic compute wrapper didn't have the interface. but it's the same if checking inside of `dense_int8`?

Reply:
I see, that works too.